sdk/core/core-amqp/test/requestResponse.spec.ts

```diff
@@ -234,19 +158,22 @@ describe("RequestResponseLink", function() {
           body: "Hello World!!"
         }
       });
-    }, 4000);
+    }, 2000);
     try {
       const controller = new AbortController();
-      const signal = controller.signal;
-      setTimeout(controller.abort.bind(controller), 100);
-      await link.sendRequest(request, {
-        delayInSeconds: 1,
-        timeoutInSeconds: 5,
-        abortSignal: signal // cancel between request attempts
-      });
+      const signal: AbortSignalLike = controller.signal;
+      const onAborted = () => {
+        // do nothing and just to mark signal as aborted internally
+      };
+      signal.addEventListener("abort", onAborted);
+      controller.abort();
+      await link.sendRequest(request, { abortSignal: signal, requestName: "foo" });
       throw new Error(`Test failure`);
     } catch (err) {
-      const expectedErrorRegex = new RegExp(/The operation has been cancelled by the user.$/, "gi");
+      const expectedErrorRegex = new RegExp(
+        /The foo operation has been cancelled by the user.$/,
+        "gi"
+      );
```

Comment:
@chradek Do you recall why we didnt add check on the error name here?

Reply:
I'm not sure why it wasn't added before, but since we are using `AbortError` from `@azure/abort-controller` everywhere now we should make sure the name matches.